### ISSUE TITLE
Datagrid filter tooltip has low contrast for accessibility

### ISSUE THREAD

**Issue by juanjosegutierrez** [vmware/clarity#6479]
## Describe the bug

The datagrid component can have a filter tooltip with a text field. The bottom border color for this text field has very low contrast compared to the background color. Currently the color contrast ratio is 2 to 1. Web Content Accessibility Guidelines states that the minimum contrast ratio should be 3 to 1.

## How to reproduce

Here is the link to the stackblitz that reproduces this: https://stackblitz.com/edit/clarity-light-theme-clr12-cds5-qv99at?file=src/app/app.component.html

Also, here is the screenshot of the stackblitz: 
<img width="552" alt="Screen Shot 2021-11-18 at 3 32 02 PM" src="https://user-images.githubusercontent.com/19143833/142513366-f63eb09f-e49d-46b6-82e0-464c32c75e2c.png">

1. Go to the stackblitz link
2. Click the filter button on the datagrid column
3. Type anything in the text field
4. Click outside the text field but inside the tooltip
5. Notice the bottom border is hard to see for accessibility users

## Expected behavior

The color contrast should be a minimum of 3 to 1 between the bottom border and the background color for the text field in the filter tooltip.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 12

**Device:**

- Type: MacBook
- OS: MacOS
- Browser: chrome
- Version: 95

## Additional notes

This issue may be present in other text fields in clarity.

**Comment by bdryanovski** [comment]
@Jinnie are we tracking this somewhere?

I know that we made some changes on the `border-color` for inputs a long time ago.

**Comment by gnomeontherun** [comment]
The work @Shijir is doing will help address this since it will use the Core colors in Clarity Angular 13.

**Comment by Jinnie** [comment]
This is a simple input element, with the color contrast of any other input element in the framework.
I don't know of any outstanding a11y bugs related to the contrast of the input field border.
I also remember contrast ratios for borders were different than these for texts.
CC @scroniser

**Comment by scroniser** [comment]
@Jinnie - There wasn't an internal issue, but I've created one based off of this issue. As far as I can tell, the issue still exists.

**Comment by gnomeontherun** [comment]
Should be resolved by #6517

**Comment by Jinnie** [comment]
Closing in favor of the internally tracked issue and the Core shim. Please see the PR linked above for more details on the shim.
